docs/gui.rst

```diff
@@ -313,6 +313,12 @@ A *event filter* is a list combined of *key*, *type* and *(type, key)* tuple, e.
 Image I/O
 ---------
 
+.. function:: gui.get_image()
+
+    :return a ``np.ndarray`` which is the current image shown on the GUI.
```

Comment:
Can we emphersize that the returned array is a RGBA array?

Reply:
Yes, good.

RGBA is guranteed by the type of `canvas::img` `Array2d<Vector4d>`, right?